index.html

```diff
@@ -5,8 +5,150 @@
+      </a>
+      
+      <nav>
+        <ul>
+          <li><a href="#">Студия</a></li>
+          <li><a href="#">Клиенты</a></li>
+          <li><a href="catalog.html">Магазин</a></li>
+          <li><a href="#">Контакты</a></li>
+        </ul>
+        <img src="https://via.placeholder.com/15x11" width="15" height="11" alt=""><!--icon корзина-->
+        <a href="#">Корзина</a>
+      </nav>
 
+      <section>
+        <h1>Долго, дорого, скрупулезно.</h1>
+        <p>Математически выверенный дизайн<br>
+          для вашего сайта или мобильного приложения.</p>
+        <a href="#">Узнать больше</a>
+      </section>
+
+      <img src="https://via.placeholder.com/760x431" width="760" height="431" alt="">
+        <input type="radio" name="kadoves" id="slaid1" checked>
+        <input type="radio" name="kadoves" id="slaid2">
+        <input type="radio" name="kadoves" id="slaid3">
+
+    </header>
+
+    <main>
+
+      <section>
+        <ul>
+          <li>
+            <img src="https://via.placeholder.com/300x146" width="300" height="146" alt="">
+              <h3>Веб-сайты</h3>
+                <p>Мир никогда не будет прежним после того как увидит ваш сайт!</p>
+                  <a href="#">Заказать</a>
+          </li>
+          <li>
+            <img src="https://via.placeholder.com/300x146" width="300" height="146" alt="">
+              <h3>Приложения</h3>
+                <p>Покорите топ-10 приложений в AppStore и Google Play</p>
+                  <a href="#">Заказать</a>
+          </li>
+          <li>
+            <img src="https://via.placeholder.com/300x146" width="300" height="146" alt="">
+              <h3>Презентация</h3>
+                <p>Вы даже не подозреваете, насколько вы изумительны!</p>
+                  <a href="#">Заказать</a>
+          </li>
+        </ul>
+      </section>
+
+    <hr>
+
+        <section>
+          <h2>Мы — маленькая, но гордая<br> дизайн-студия из Краснодара.</h2>
+            <p>Исповедуем принципы минимализма и чистоты. Ставим математический расчет<br>
+              превыше креатива. Работаем не покладая рук, как роботы.</p>
+          <h3>Выполняем заказы на разработку:</h3>
+              <ul>
+                <li>Веб-сайтов любой сложности</li>
+                <li>Мобильных приложений для iOS и Android</li>
+                <li>Слайдшоу и видео для корпоративных презентаций</li>
+              </ul>
+        </section>
+
+        <section>
+          <img src="https://via.placeholder.com/360x208" width="360" height="208" alt="">
+            <table>
+              <caption>с 2004 года Любим точность во всем:</caption>
+                <tr>
+                  <td>146%</td>
+                  <td>100%</td>
+                  <td>50%</td>
+                </tr>
+                <tr>
+                  <td>Уровень самоотдачи</td>
+                  <td>Соблюдение сроков</td>
+                  <td>Минимальная предоплата</td>
+                <tr
```

Comment:
```suggestion
                </tr>
```
Должен быть закрывающий тег.